guides/building_api.md

```diff
@@ -14,6 +14,7 @@ API is an important part of web applications. They provide interfaces for commun
 * Each API resource class (e.g. `UsersResource`) defines a single resource (e.g. `resource :users do ... end`).
 * Entities are stored in `entities/` directory and inherited from `Grape::Entity` class.
 * Each API resource, entity class or helpers module class are wrapped into corresponding module (e.g. `AppV1API::Resources`, `AppV1API::Entities`, `AppV1API::Helpers`).
+* For duplicate params across the project use special `SharedParamsHelper` helper.
```

Comment:
1. This sentence is not very English ;)
2. It is useful for sharing params between several API resources. If params are shared between "actions" of the same resources then it is better to keep them inside the resource.

Probbly, something like this will sound a bit better:
"`SharedParamsHelper` helper may be used to share params between resources."